src/main/java/com/puppycrawl/tools/checkstyle/checks/design/MutableExceptionCheck.java

```diff
@@ -66,6 +66,44 @@
  * <pre>
  * &lt;module name=&quot;MutableException&quot;/&gt;
  * </pre>
+ * <p>Example:</p>
```

Comment:
Thanks a lot, please create new config example for other properties of Check.
Please use same java code as to have now, and only trailing comments of ok or violation should change

Reply:
Do you mean with the "error" keyword in the classname and by extending the error class, right ?

Reply:
https://checkstyle.org/config_design.html#MutableException_Properties

you need to change `format` and `extendedClassNameFormat` to show user how chnages in it make validation different.